packages/expo-image-manipulator/ios/ImageManipulatorErrors.swift

```diff
@@ -0,0 +1,86 @@
+// Copyright 2021-present 650 Industries. All rights reserved.
+
+import CoreGraphics
+import ExpoModulesCore
+
+internal struct ImageContextLostError: CodedError {
+  var description: String {
+    "Image context has been lost."
+  }
+}
```

Comment:
Well, it's not the same actually. Tbh I still haven't decided what's the final shape of errors — it will need some adjustments, especially that I'd really like to be able to chain errors (something like "caused by" in JVM, also see [ECMAScript proposal](https://github.com/tc39/proposal-error-cause))